code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -358,6 +358,12 @@
 /datum/reagent/medicine/styptic_powder/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message = 1)
 	if(iscarbon(M) && M.stat != DEAD)
 		if(method in list(INGEST, VAPOR, INJECT))
+			if (method == TOUCH)
```

Comment:
This will literally never proc because you ask if method is touch under condition that method is either ingest, vapor or inject.

Reply:
Also touch is pretty much the only method meant for it. I don't see why you should be *breaking* this chemical, instead you could've nerfed or hard-capped the healing.

Reply:
only works with beakers